### ISSUE TITLE
Pressing 'Restart' in quick succession should not be possible.

### ISSUE THREAD

**Issue by Yakehira** [x64dbg/x64dbg#2333]

**Comment by torusrxxx** [comment]
Yes, it is indeed very slow, and it is how the developers trigger and test some crash bugs of x64dbg due to race condition. People shouldn't do these things themselves.

**Comment by czghzc** [comment]
It is really slow when restarting, it is recommended to keep the information in the debugger when restarting and only restart the process

**Comment by torusrxxx** [comment]
If you want to suggest a feature to overwrite the original with the patched version, open a separate issue please.
